### PR TITLE
Blog: Getting Started With Software Factory And Zuul3

### PR DESCRIPTION
**Pull request by morucci**

### CODE REVIEW

**Comment by dmsimard** [comment]
@morucci considering this is on the RDO blog, can we at least mention that RDO actually uses software factory for review.rdoproject.org ? :)

File: source/blog/2017-11-28-getting-started-with-software-factory-and-zuul3.html.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Then you should see the jobs being executed on the ZuulV3 status page.
+
+![Zuul buildset](images/zuul.png)
+
+And get the jobs' results on the corresponding Gerrit review page.
+
+![Gerrit change](images/gerrit.png)
+
+Finally, you should find the links to the generated artifacts and the ARA reports.
```
Comment (dmsimard) [inline source/blog/2017-11-28-getting-started-with-software-factory-and-zuul3.html.md]: I'd appreciate if you could put a link for ARA: https://github.com/openstack/ara

Thanks !

**Comment by morucci** [comment]
@dmsimard Thanks for the review I pushed a new commit to add changes according to your comments. Let me know if changes are ok for you.

**Comment by rbowen** [comment]
+1. 
Looks good to me.

**Comment by dmsimard** [comment]
lgtm

**Pull request merged**